tests/countries/test_french_polynesia.py

```
@@ -12,20 +12,14 @@
 
 from unittest import TestCase
 
-from holidays.countries.french_polynesia import HolidaysPF, FrenchPolynesia, PF, PYF
+from holidays.countries.french_polynesia import FrenchPolynesia
 from tests.common import CommonCountryTests
 
 
 class TestFrenchPolynesia(CommonCountryTests, TestCase):
     @classmethod
     def setUpClass(cls):
```

Ruff (0.14.3)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)